static/security.txt

```diff
@@ -0,0 +1,4 @@
+Contact: mailto:security@roadie.io
+Expires: 2022-06-21T11:00:00.000Z
+Preferred-Languages: en,es
```

Comment:
:smile:

Reply:
😉  I should have added Finnish and Bosnian but it's too late now.